netsim/devices/cumulus.py

```diff
@@ -15,10 +15,24 @@ def check_ospf_vrf_default(node: Box) -> None:
         category=log.IncorrectType,
         module='quirks')
 
+"""
+Checks whether any node VLAN is in the (changeable) reserved range
+"""
+def check_reserved_vlan_range(node: Box) -> None:
```

Comment:
Why is this change in a Cumulus NVUE VLAN PR?

Reply:
Removed for now, may revisit (more generally?) in the future